xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-rss/src/main/java/org/xwiki/rendering/internal/macro/rss/RssMacro.java

```diff
@@ -128,21 +130,16 @@ public boolean supportsInlineMode()
         SyndFeed feed = this.romeFeedFactory.createFeed(parameters);
 
         if (parameters.isDecoration()) {
-            BoxMacroParameters boxParameters = new BoxMacroParameters();
             boolean hasImage = parameters.isImage() && (feed.getImage() != null);
-            boxParameters.setCssClass(FEED_CLASS_VALUE);
+            parameters.setCssClass(FEED_CLASS_VALUE);
```

Comment:
Same comment I gave for AbstractMessageMacro. In general you should always avoid modifying what you get as input unless you are officially supposed to do it in the design of the API.

Reply:
I tried to override getClassProperty but I obtained failing test cases, so I keep things simple as I mostly wanted to fix the tests.